server/src/repositories/teams.repository.ts

```diff
@@ -105,4 +105,19 @@ export class TeamRepository extends Repository<Team> {
 
 		return this.save(team);
 	}
+
+	async removeUserFromTeam(userId: string, teamId: string) {
+		const team = await this.createQueryBuilder('Team')
+			.where('Team.id = :id', { id: teamId })
+			.leftJoinAndSelect('Team.users', 'Users')
+			.getOne();
```

Comment:
this.findOne

Reply:
I suppose it is not required, but just a matter of opinion )

Reply:
this.findOne is cleaner